.github/workflows/test_and_deploy.yml

```diff
@@ -7,24 +7,7 @@ env:
   IMAGE_NAME: ${{ github.repository }}
   
 jobs:
```

Comment:
See question on `cellfinder-core`, can we keep the lint -> test -> deploy dependency?

Reply:
yep will look into this on both